INSTALL.md

```diff
@@ -3,19 +3,16 @@
 ## Prerequisites
 
 ### Hardware
-1. A Raspberry Pi or similar SBC capable of running Linux, with Bluetooth antenna.
-2. Micro SD card + SD adapter + necessary cables for OS installation (not detailed in this procedure).
-
-
-> [!Important]
-> In what follows, "RPi" refers to the device where the service will be executed.
+1. A Linux based computer which could be the same machine that runs Home Assistant (HA), or a seperate device located remotely from the HA machine, but close to where the car is located. We'll call this device the Host Device (HD). A Raspberry Pi makes an ideal HD
+2. Bluetooth Low Energy (BLE) capabilities. This could be a plug in USB device or built-in to the HD. A strong BLE signal is required for reliable operation, so the HD should be located close to where the car is normally parked
+3. An existing Linux OS installation which is already installed and operational on the HD. For a RPi this could be DietPi or Raspberry Pi OS. You will need to check that BLE (not just standard Bluetooth without BLE) is operational before proceeding
 
 ### Software
-#### On the RPi
-1. A Linux distribution (DietPi used for this procedure). **Remember to enable Bluetooth in the settings!**
-2. Docker
-3. Docker-compose
-4. Optional: Portainer, useful for viewing logs and other details.
+#### On the Host Device
+1. A Linux distribution
+2. docker. As an example for a Raspberry Pi OS 32-bit see these instructions https://docs.docker.com/engine/install/raspberry-pi-os/
+3. docker-compose. Not required if using Portainer. If using Raspberry Pi OS use the following command to ensure you get the correct version: `sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin docker-compose`

```

Comment:
FYI the latest Docker doesn't come with `docker-compose` but `docker compose` - Your Linux distribution if old will have `docker-compose` available but not the latest one. I think it comes with docker-ce-cli...

Actually, if you try `docker compose` I bet it might work. If so, I propose we remove that package `docker-compose` and update the doc to use `docker compose` instead of `docker-compose`.  The later is officially obsolete since ~2-3y and I'm afraid latest distribution just don't have that pkg available.